src/welcome/welcome.window.js

```diff
@@ -26,17 +28,26 @@ function createWelcomeWindow() {
 		useContentSize: true,
 		webPreferences: {
 			preload: WELCOME_WINDOW_PRELOAD_WEBPACK_ENTRY,
+			zoomFactor,
 		},
 		icon: getBrowserWindowIcon(),
 	})
 
+	// Hide traffic light buttons on Mac
 	if (isMac()) {
-		// Hide traffic light buttons on Mac
 		window.setWindowButtonVisibility(false)
 	}
 
 	window.loadURL(WELCOME_WINDOW_WEBPACK_ENTRY)
 
+	// Setting zoomFactor with webPreferences is not enough.
+	// If is only applied if it was set after the window is ready.
+	// Affects all the following windows.
```

Comment:
This sounds weird

Reply:
The electron's bug or this text specifically?

Reply:
the last line is not a real sentence :see_no_evil: